pyuvdata/uvdata.py

```diff
@@ -1698,6 +1701,66 @@ def write_miriad(self, filepath, run_check=True, check_extra=True,
                                 clobber=clobber, no_antnums=no_antnums)
         del(miriad_obj)
 
+    def read_uvh5(self, filename, run_check=True, check_extra=True,
+                  run_check_acceptability=True):
+        """
+        Read a UVH5 file.
```

Comment:
Since this is our first attempt at splitting that previously assumed equality and there is at least one other existing hdf5 format we need to make super clear that UVH5 is "a pyuvdata object stored in an HDF5 container".  That sentence should be plastered everywhere, in all the major functions. That way people trying to figure out why they can't read their random hdf5 file and drop into one of the read functions will get the message.

Reply:
Okay, I can make this clearer when I update the documentation.